src/features/surveys/actions/submit.ts

```diff
@@ -0,0 +1,23 @@
+'use server';
+
+import BackendApiClient from 'core/api/client/BackendApiClient';
+import prepareSurveyApiSubmission from 'features/surveys/utils/prepareSurveyApiSubmission';
+import { ZetkinSurveyFormStatus } from 'utils/types/zetkin';
+
+export async function submit(
+  prevState: ZetkinSurveyFormStatus,
+  formData: FormData
+): Promise<ZetkinSurveyFormStatus> {
+  const data = Object.fromEntries([...formData.entries()]);
+  const submission = prepareSurveyApiSubmission(data, false);
+  const apiClient = new BackendApiClient({});
```

Comment:
Why are headers not being forwarded to the `BackendApiClient()` constructor? Unless I'm mistaken, this means that the user will not be authenticated when making this request, which means the "user" signature type can never work.

Reply:
Fixed!